app.rb

```diff
@@ -52,81 +65,127 @@ def create_person
   end
 
   def create_student
-    puts 'create a new student'
+    puts 'Create a new student'
     print 'Enter student age: '
     age = gets.chomp.to_i
     print 'Enter name: '
     name = gets.chomp
     print 'Has parent permission? [y/n]: '
     parent_permission = gets.chomp.downcase
-    case parent_permission
-    when 'n'
-      student = Student.new(age: age, name: name, parent_permission: parent_permission, classroom: @classroom)
-      @persons << student
-      puts 'Student doesn\'t have parent permission, can\'t rent books'
-    when 'y'
-      student = Student.new(age: age, name: name, parent_permission: parent_permission, classroom: @classroom)
-      @persons << student
-      puts 'Student created successfully'
+
+    student = Student.new(age: age, name: name, parent_permission: parent_permission, classroom: @classroom)
+    @persons << student
+
+    if parent_permission == 'n'
+      puts "Student #{name} doesn't have parent permission and can't rent books."
+    elsif parent_permission == 'y'
+      data = @preserve_person
+      @persons.each do |person|
+        data << { name: person.name, id: person.id, age: person.age, classname: person.class.name }
+      end
+      Store.save('people.json', data)
+      puts 'Student created successfully.'
+    else
+      puts 'Invalid input for parent permission. Try again.'
     end
   end
 
   def create_teacher
-    puts 'create a new teacher'
+    puts 'Create a new teacher'
     print 'Enter teacher age: '
     age = gets.chomp.to_i
     print 'Enter teacher specialization: '
     specialization = gets.chomp
     print 'Enter teacher name: '
     name = gets.chomp
```

Comment:
### Optional
- [ ] It would be nice if we fix the bug in the app, here to list a rental given an id does not work as expected. Here is a screenshot to show it
![Screenshot from 2023-06-29 03-28-42](https://github.com/fredo509/OOP-school-library/assets/46388113/1c09242f-6727-4679-b765-72a90eebdee5)
Although i entered the data as instructed below as seen
![Screenshot from 2023-06-29 03-28-09](https://github.com/fredo509/OOP-school-library/assets/46388113/8c48d1cf-d227-4e37-883e-2e26d5cc97b7)
![Screenshot from 2023-06-29 03-27-48](https://github.com/fredo509/OOP-school-library/assets/46388113/032e5879-31ca-426f-8f3a-add479891183)